db/migrate/20150110084027_change_team_slug_to_citext.rb

```diff
@@ -0,0 +1,5 @@
+class ChangeTeamSlugToCitext < ActiveRecord::Migration
+  def change
+    change_column :teams, :slug, :citext, null: false, index: true , unique: true
```

Comment:
Doesn't seem to happen on my local. The column gets changed, but the index isn't added. Could you verify whether the index has been added in production? 

Also, a small note: The `schema.rb` was not updated after you changed the index to unique. No big deal, it should get rewritten whenever the next migration is written :)